tests/unit/conftest.py

```diff
@@ -242,3 +243,10 @@ def dummy_empty_response():
             "hits": []
         }
     }
+
+
+@pytest.yield_fixture
+def httppretty_mock():
+    httpretty.enable()
+    yield
+    httpretty.disable()
```

Comment:
The only nitpick I could really find is that this fixture could have used a comment explaining that the decorator doesn't work with pytest, but who cares.